src/Symfony/Component/ExpressionLanguage/ExpressionLanguage.php

```diff
@@ -103,6 +103,11 @@ public function parse($expression, $names)
         return $parsedExpression;
     }
 
+    public function dump(ParsedExpression $expression)
+    {
+        return $expression->getNodes()->dump();
+    }
```

Comment:
I would remove this method and add a `dump()` method on `ParsedExpression` instead.